src/main/resources/application.properties

```diff
@@ -75,7 +75,7 @@ spring.mongodb.embedded.version: 3.4.1
 # starts and activate it manually in tests where we need the Spring's own embedded mongo DB
 spring.autoconfigure.exclude: org.springframework.boot.autoconfigure.mongo.embedded.EmbeddedMongoAutoConfiguration
 
-threads.core.pool.size: 100
-threads.queue.capacity: 5000
-threads.max.pool.size: 150
-scheduled.threadpool.size: 100
+threads.core.pool.size: 200
+threads.queue.capacity: 7000
+threads.max.pool.size: 250
+scheduled.threadpool.size: 200
```

Comment:
Could you elaborate on where you got these values, you increased them but why are the new values good?

Reply:
Failover tests and some other tests fails if we don't use specific size thread pool. I don't know why, seem to be timing issues when things should be completed and not completed in time. I don't know tests and details of the tests, so can't answer why. Maybe those that wrote the tests can answer your question.
Since I removed the unlimit number of threads, the size of the single thread pool can be increased also.

Reply:
But where did you get the values?

Reply:
change the values until tests succeeds, same as it has been done earlier I guess.
This question has been raised earlier, no one have given any better answer.

Reply:
When application is deployed in production, user can set other values.